GitExtensionsShellEx/GitExtensionsShellEx.vcxproj

```diff
@@ -115,7 +115,7 @@
       <BrowseInformation>true</BrowseInformation>
       <WarningLevel>Level3</WarningLevel>
       <SuppressStartupBanner>true</SuppressStartupBanner>
-      <DebugInformationFormat>EditAndContinue</DebugInformationFormat>
```

Comment:
should we enable increament build instead of disable edit and cont?

Reply:
dont mind if cannot

Reply:
Had to change /SAFESEH too. This was only for debug builds, so not even visible in Appveyor (but nice to have no warnings in local build too).

Reply:
This was changed for Debug.
Thanks!